tests/behat/features/cu_advanced_design_bundle/blocktitleicons.feature

```diff
@@ -0,0 +1,28 @@
+@AdvDesign
+Feature: Font Awesome Icons in block titles
+In order to add visual interest to my site
+As an authenticated user
+I should be able to attach Font Awesome icons to my block titles
+
+# TESTING NOTE: ATTACHING A BLOCK TO A PAGE IS VERY DIFFICULT WITH BEHAT; 
+# IT CAN BE DONE BUT THE TEST IS NOT ROBUST; SO IT HAS TO BE A FUNCTIONAL TEST, NOT AN AUTOMATED ONE
+
+@javascript
+Scenario: An authenticated user can add an icon to a block title
+Given I am logged in as a user with the "site_owner" role
+
+# CREATE A TEXT BLOCK AND ATTACH AN ICON TO IT
+And I am on "block/add/block"
+And I fill in "edit-label" with "Apple Info Block"
+And fill in "edit-title" with "Apple Info Block"
+And I follow "Disable rich-text"
+And I fill in "Body" with "A is for Apple"
+And I press "Save"
+And I go to "block/apple-info-block/design"
+# SEE IF FOLLOWING BLOCK DESIGNER WORKS
+# And I follow "Block Designer" NO IT DOESN'T WORK; CAN'T BE FOUND
+And I click the "#edit-icon a.fieldset-title" element
+And I select "fa-apple" from "exbd_icon"
+And I press "edit-submit"
+#YES THE FOLLOWING LINE IS MISSPELLED IN WEB EXPRESS
+Then I should see "Block Desiger settings have been saved."
```

Comment:
```gherkin
Scenario: An authenticated user can add an icon to a block title
Given I am logged in as a user with the "site_owner" role

# CREATE A TEXT BLOCK AND ATTACH AN ICON TO IT
And I am on "block/add/block"
And I fill in "Label" with "Apple Info Block"
And fill in "Title" with "Apple Info Block"
#And I follow "Disable rich-text"
And I fill in "Body" with "A is for Apple"
And I press "Save"
# And I go to "block/apple-info-block/design"
# SEE IF FOLLOWING BLOCK DESIGNER WORKS
And I follow "Block Designer"
#And I click the "#edit-icon a.fieldset-title" element
And I select "fa-apple" from "exbd_icon"
And I press "Save"
#YES THE FOLLOWING LINE IS MISSPELLED IN WEB EXPRESS
Then I should see "Block Desiger settings have been saved."
```

That works for me without the `@javascript` tag which is faster to run. It also uses the labels for fields instead of the CSS which is more readable. It would be nice (and I think more accessible?) if the radios had labels but they don't.

Reply:
For radio buttons, I've found that you have to select  "radio-button-value" (NOT ID) from "radio-button-group-name". And yes, since the test is less complex, it doesn't need the javascript tag any more.